.circleci/config.yml

```diff
@@ -205,7 +205,17 @@ jobs:
       - run:
           name: Execute Integration Tests
           command: |
-            yarn run test:integration
+            run_int_tests() {
```

Comment:
Why are we not using docker-compose to start up the handlers similar to how we do so with the api-svc?

I believe everything should work as expected if we used the existing mechanism.

Reply:
running all services altogether breaks integration tests.
services start to communicate with each other, and test consumers get only a portion of all messages